pykrige/ok.py

```diff
@@ -1,811 +1,850 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals

```

Comment:
I do not understand what's wrong with the GitHub diff here. These `__future__` import lines have not changed and yet github indicated that they did. Maybe something about the line break character? 

In any case we need to resolve this as otherwise it's quite hard to review.

Reply:
Rebasing resolved this. Diffs are now clearer.